src/esp/sensor/PinholeCamera.cpp

```diff
@@ -110,6 +110,10 @@ void PinholeCamera::drawObservation(sim::Simulator& sim) {
     // TODO: check sim has semantic scene graph
     renderer->draw(*this, sim.getActiveSemanticSceneGraph(),
                    sim.isFrustumCullingEnabled());
+    if (&sim.getActiveSemanticSceneGraph() != &sim.getActiveSceneGraph()) {
+      renderer->draw(*this, sim.getActiveSceneGraph(),
+                     sim.isFrustumCullingEnabled(), true);
+    }
```

Comment:
Note that this code is actually duplicated in python which calls directly into `render->draw` for sensors.